graphistry/ai/requirements.txt

```diff
@@ -5,6 +5,7 @@ scipy
 pandas
 dirty-cat
 umap-learn
+cuml
```

Comment:
Let's keep rapids out of our ai deps

- it's not installable in pip (I think)
- a massive & flaky  conda dep

So would break pygraphistry[ai] install in most cases

If do we want it as an optional dep, would seperate it out  more like `graphistry[rapids]`